libs/detectors/detector.py

```diff
@@ -11,11 +13,10 @@ def __init__(self, config, source):
         self.config = config
         self.device = self.config.get_section_dict("Detector")["Device"]
         self.resolution = tuple([int(i) for i in self.config.get_section_dict("App")["Resolution"].split(",")])
-        # TODO: Iterator in the below line should be replaced with:
-        #  get_model_json_file_or_return_default_values(self.config, self.device, self.config.get_section_dict(source)["Id"])["variables"]["ImageSize"]
-        self.image_size = [int(i) for i in self.config.get_section_dict("Detector")["ImageSize"].split(",")]
-        # Do: Ctrl + Shift + F and type: "ImageSize" and "get_section_dict("Detector")" (also with this '' quotes)
-
+        self.image_size = [int(i) for i in get_model_json_file_or_return_default_values(self.config, self.device, self.config.get_section_dict(source)["Id"])["variables"]["ImageSize"].split(",")]
+        logger.info("-----------------------------------------")
+        logger.info(source)
+        logger.info("-----------------------------------------")
```

Comment:
will these stay? If so I'd remove logging "-----" and adding a message before the  source info

Reply:
No, should be deleted.